setup.py

```diff
@@ -0,0 +1,5 @@
+from distutils.core import setup
+setup(name='kellerLD',
+      version='1.0',
+      py_modules=['kellerLD'],
+      )
```

Comment:
Ita missing python-smbus as a dependency

Reply:
This has now been added to the setup.py